android/sentry.properties

```diff
@@ -0,0 +1,4 @@
+defaults.url=https://sentry.io/
+defaults.org=intellisys-d-corp
+defaults.project=covid-rd
+auth.token=180e3423a85742a1bd3f245db8e666fcb693fd43c6a64c168eb320833b6afc74
```

Comment:
Again, is it right to have tokens and keys directly into the code?

Reply:
Addressed